apps/financial-aid/web-osk/src/utils/useNavigationTree.ts

```diff
@@ -1,6 +1,4 @@
-import { useEffect, useState } from 'react'
-
-const useNavigationTree = () => {
+const useNavigationTree = (hasIncome: boolean | undefined) => {
```

Comment:
This should not take in undefined. Handle it out side of this.

Reply:
hasIncome is undefined until reach upon the site where you answer your income situation, should hasIncome be default false then?

Reply:
The current implementation gives the same result for false and undefined. We just don't want this complexity to bleed into this function. You can just use Boolean(form.hasIncome) when you use `useNavigationTree`.

Reply:
Also the `FormStepperSection` type should be part of this file, not the `useFormNavigation`.